Change logging config to logging.config.dictConfig

JPrevost opened the pull request:
closes #97

gravesm commented on kepler/app.py:
```diff
@@ -45,20 +46,6 @@ def handle_unsupported_format(error):
-    requests_log = logging.getLogger('requests.packages.urllib3')
-    requests_log.setLevel(logging.DEBUG)
-    requests_log.addHandler(handler)
+    with open('kepler/logging.conf') as f:
```
We should pull the location of the configuration file from an environment variable. I'm fine with having it default to this location, but we may want to be able to use a different config for testing, for example.

JPrevost replied:
Cool. Will change.

JPrevost commented:
@gravesm: d933424 addresses code review suggestions.

I considered setting `LOGGING_CONFIG` in `kepler/settings.py` to consolidate all ENV reading in one place, but decided this felt different enough since there was a default value we wanted to provide to keep it here. I could easily be swayed though so let me know if have opinions.

gravesm commented:
:heavy_check_mark:
